src/main/Validator.ts

```diff
@@ -197,22 +176,45 @@ function cleanURL(url: string): string {
     return updatedURL;
 }
 
-export function validateV2ConfigData(data: ConfigV2) {
-    if (Array.isArray(data.teams) && data.teams.length) {
-        // first replace possible backslashes with forward slashes
-        let teams = data.teams.map((team) => {
+function cleanTeam<T extends {name: string; url: string}>(team: T) {
```

Comment:
I consolidated all the functions to clean the URLs in teams and change around that logic. Made so that code coverage was increased without writing duplicate tests, and also reduces a bit of code bloat.

Reply:
I don't think it really matters one way or another, since generics allow us to just check for the fields we care about for this function. It's not like there's a large amount of in-line typing happening here, so I think it's fine doing it this way.